resources/Dockerfile

```diff
@@ -4,7 +4,7 @@
 FROM ubuntu:20.04 as dev
 
 ARG TARGETARCH
-ARG RUST_TOOLCHAIN="1.59.0"
+ARG RUST_TOOLCHAIN="1.60.0"
```

Comment:
Don't you think we should use the latest version here? (1.61)

Reply:
I want the version to match the minimum version otherwise we won't have consistent testing..

Reply:
Fair enough!